_includes/footer.html

```diff
@@ -170,24 +170,30 @@ <h2 class="h6">
                         </noscript>
                     </div>
                     <p class="footer_address">
-                        <span class="footer_address-word-break">
-                            Consumer Financial
+                        <span itemprop="name">
+                            <span class="footer_address-word-break">
+                                Consumer Financial
+                            </span>
+                            Protection Bureau<br>
+                        </span>
+                        <span itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+                            <span itemprop="streetAddress">1700 G Street, NW<br></span>
```

Comment:
@sebworks I wouldn't just yet, beta clean up is more important. I thought we were supposed to be done w/ the standards stuff yesterday and needed some real examples of Schema in use so I quickly put this together before the meeting. We could even leave this PR open and use it for updating everything with Schema next sprint.